docs/application/dotnet/guides/nui/window.md

```diff
@@ -0,0 +1,335 @@
+For the case, the preferred orientation function is supplied.
+We recommand that the preferred orientation is one of avialable orientations.
+For instance, look at the below example.
+The available orientation list has Portrait, Landscape and LandscapeInverse, then the preferred orientation is set with Landscape.
+After that, the window will only work Landscape although device is rotated with any orientation.
+
+```csharp
+// To enable available orientations
+window.AddAvailableOrientation( Portrait );
+window.AddAvailableOrientation( Landscape );
+window.AddAvailableOrientation( LandscapeInverse );
+
+// To set the preferred orientation with Landscape.
+window.SetPreferredOrientation( Landscape );
+```                                                                                                                               
+
+> **Note**
+>
+> If the Preferred Orientation's function is used without the available orientation list, the rotation will not work normal.
+>
+
+<a name="multiwindow"></a>
+## Multi Window
+
+<a name="additionalwindow"></a>
+### Creating Additional Window
+
+It is easy to create an additional window in addition to the default window. The size and position of the new window should be specified. Each window can has its own background color and title.
+
+```csharp
+Window newWindow = new Window(new Rectangle(0, 0, 1920, 1080))
+{
+    BackgroundColor = Color.White,
+    Title = "new window"
+};
+```
+
+<img src="./media/multiwindow3.png">
+
+> **Note**
+>
+> To support rotation in multi window, the [rotation](#rotation) functions should be used for each window.
+>
+
+<a name="raisinglowering"></a>
+## Changing window stacking order
+
+NUI provides support for changing the stacking order of the windows.
+
+A window can be raised to the top of the window stack so that no sibling window obscures it.
+
+```csharp
+window.Raise();
+```
+
+A window can be lowered to the bottom of the window stack so that it does not obscure any sibling windows.
+
+```csharp
+window.Lower();
+```
+
+New windows are automatically put to the top of the window stack at creation time by the window manager.
+
+
+<a name="transientfor"></a>
+### Setting parent window
+
+To set transient for two windows, SetParent function is useful.
+After setting that, these windows do together when raise-up, lower and iconified/deiconified.
+Initially, the window is located on top of the parent. The window can go below parent by calling Lower().
+```csharp
+// Get the parent window instance
+Window parent = Window.Instance;
+
+Window child = new Window(new Rectangle(0, 0, 960, 1080))
+{
+    BackgroundColor = Color.White,
+    Title = "child window"
+};
+
+child.SetParent( parent );
+```
+
+To unset transient for relationship, call Unparent().
+```csharp
+child.Unparent();
+```
+
+> **Note**
+>
+> If parent's window stack is changed by calling Raise() or Lower(),> child windows are located on top of the parent again.
+>
+
```

Comment:
Please add "Related information" section.

## Related Information
- Dependencies
  - Tizen 5.5 and Higher